wiki/osu!_rank_one_history/en.md

```diff
@@ -0,0 +1,275 @@
+# History of osu! Rank 1 Players
+
+The following is a list of players who are recognized to have reached rank 1 in the [osu! Global Rankings](https://osu.ppy.sh/rankings/osu/performance).
+
+Over the years the osu! global rankings has used three different systems for ranking players:
+
+- [Ranked Score](/wiki/score#ranked-score) (July 2007 - April 2012)
+- [ppv1](/wiki/Performance_points/ppv1) (April 2012 - 27 January 2014)
+- [ppv2](/wiki/Performance_points) (27 January 2014 - Present)
+Because of this, this list will only include players who reached rank 1 on the main leaderboard that was used at the time of reaching said rank.
```

Comment:
```suggestion
- [Ranked Score](/wiki/score#ranked-score) (July 2007 - April 2012)
- [ppv1](/wiki/Performance_points/ppv1) (April 2012 - 27 January 2014)
- [ppv2](/wiki/Performance_points) (27 January 2014 - Present)

Because of this, this list will only include players who reached rank 1 on the main leaderboard that was used at the time of reaching said rank.
```
asc likes empty lines separating paragraphs